stage0/run.go

```diff
@@ -443,12 +455,12 @@ func prepareAppImage(cfg PrepareConfig, appName types.ACName, img types.Hash, cd
 			}
 		}
 
-		if err := ioutil.WriteFile(common.AppTreeStoreIDPath(cdir, appName), []byte(treeStoreID), 0700); err != nil {
+		if err := ioutil.WriteFile(common.AppTreeStoreIDPath(cdir, appName), []byte(treeStoreID), defaultRegularFilePerm); err != nil {
 			return fmt.Errorf("error writing app treeStoreID: %v", err)
 		}
 	} else {
 		ad := common.AppPath(cdir, appName)
-		err := os.MkdirAll(ad, 0755)
+		err := os.MkdirAll(ad, defaultRegularDirPerm)
```

Comment:
@yifan-gu 

> I think we can chown here like what we do in rendering stage1.

Ok. It seems to work fine with the uid shift that you do below :+1:

While checking the permissions of the rootfs, I found this bug appc/docker2aci#94.

> Not sure what if we enable private users with overlay later...

We can worry about this later: private users with overlay would require a kernel API that does not exist yet (#1057).